cmd/notation/sign_test.go

```diff
@@ -154,7 +159,9 @@ func TestSignCommmand_OnDemandKeyOptions(t *testing.T) {
 		"-u", expected.Username,
 		"--password", expected.Password,
 		"--id", expected.KeyID,
-		"--plugin", expected.PluginName}); err != nil {
+		"--plugin", expected.PluginName,
+		"--force-referrers-tag=false",
```

Comment:
Why are we making this change? Also, applies to below tests

Reply:
> Why are we making this change? Also, applies to below tests

Oh, it's just an extra check, having `--force-referrers-tag=false` is equivalent to not setting the `--force-referrers-tag` flag at all.